reccmp/assets/reccmp.js

```diff
@@ -144,6 +144,15 @@ function copyToClipboard(value) {
   navigator.clipboard.writeText(value);
 }
 
+function escapeHtml(unsafe) {
+  return unsafe
+    .replace(/&/g, '&amp;')
+    .replace(/</g, '&lt;')
+    .replace(/>/g, '&gt;')
+    .replace(/"/g, '&quot;')
+    .replace(/'/g, '&#039;');
+}
```

Comment:
How about building the DOM using the API and setting the string using `.textContent` as suggested in https://stackoverflow.com/a/66481918 (probably where this code came from)? We can restrict the refactor to the critical parts for now.